app/absentee/templates/absentee/email/file_notification.html

```diff
@@ -49,7 +49,13 @@
 
 Helpful hints:
 <ul style="margin-left:0;padding-left:0">
-    <li>If you don’t have a printer, you can print the form out at any local library.</li>
+    <li>
+        If you don’t have a printer, you can print the form out at any local library. If your
+        library isn't open, you can print for a few dollars at
+        <a href="https://www.staples.com/services/printing/" target="_blank">Staples</a>,
+        <a href="https://www.fedex.com/en-us/office.html" target="_blank">FedEx Office</a>,
+        or <a href="https://www.theupsstore.com/print" target="_blank">UPS</a>.
```

Comment:
```suggestion
        <a href="https://www.staples.com/services/printing/" target="_blank" rel="noopener noreferrer">Staples</a>,
        <a href="https://www.fedex.com/en-us/office.html" target="_blank" rel="noopener noreferrer">FedEx Office</a>,
        or <a href="https://www.theupsstore.com/print" target="_blank" rel="noopener noreferrer">UPS</a>.
```

re: https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

I usually lint this in ember-template-lint so not sure where in React yet

Reply:
Not super relevant for email (neither is `target="_blank"` to be honest)